src/javascripts/services/accountService.js

```diff
@@ -29,8 +30,12 @@ class AccountService{
     * @returns {Boolean} True if the account is valid, otherwise false.
     */
     isValidAccount = ({email, password}) => {
-        const validAccount = this.accountList.some((item) => item.email === email && item.password === password)
-        return validAccount;
+        const isAccountValid = this.accountList.some((item) => item.email === email && item.password === password)
```

Comment:
This function for check valid account. Should NOT do other thing here. Like saveLocalStorage

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks!